.gitignore

```diff
@@ -10,4 +10,4 @@ target/
 .idea
 
 # Ignore API keys
-keys/
\ No newline at end of file
+keys/meltingpot-step2020-91a60ee70597.json
```

Comment:
Honestly I would keep the key out of the repo folder entirely, less chance for error that way.

Reply:
Yeah, this is probably the best way to go about it moving forward

Reply:
good call!